specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1494 @@
+              "description": "configuration for Microsoft Defender for Server VM scanning",
+              "properties": {
+                "cloudRoleArn": {
+                  "type": "string",
+                  "description": "The cloud role ARN in AWS for this feature"
+                },
+                "scanningMode": {
+                  "type": "string",
+                  "description": "The scanning mode for the VM scan.",
+                  "enum": [
+                    "Default"
+                  ],
+                  "x-ms-enum": {
+                    "name": "scanningMode",
+                    "modelAsString": true
+                  }
+                },
+                "exclusionTags": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "string"
+                  },
+                  "description": "VM tags that indicates that VM should not be scanned"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "defenderFoDatabasesAwsOffering": {
+      "type": "object",
+      "description": "The Defender for Databases AWS offering",
+      "x-ms-discriminator-value": "DefenderForDatabasesAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "arcAutoProvisioning": {
+          "type": "object",
+          "description": "The ARC autoprovisioning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is arc auto provisioning enabled"
+            },
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS for this feature"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "Configuration for servers Arc auto provisioning",
+              "properties": {
+                "proxy": {
+                  "type": "string",
+                  "description": "Optional http proxy endpoint to use for the Arc agent"
+                },
+                "privateLinkScope": {
+                  "type": "string",
+                  "description": "Optional Arc private link scope resource id to link the Arc agent"
+                }
+              }
+            }
+          }
+        },
+        "rds": {
+          "type": "object",
+          "description": "The RDS configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is RDS protection enabled"
+            },
+            "cloudRoleArn": {
+              "type": "string",
+              "description": "The cloud role ARN in AWS for this feature"
+            }
+          }
+        },
+        "databasesDspm": {
+          "type": "object",
+          "description": "The databases DSPM configuration",
```

Comment:
>DSPM [](http://example.com/codeflow?start=40&length=4)

You should expand less well-known acronyms in docs, e.g. "digital security posture management (DSPM)"

Reply:
Done